composer.json

```diff
@@ -32,7 +32,7 @@
     "drupal-composer/drupal-scaffold": "^2",
     "drupal/admin_toolbar": "^1.0",
     "drupal/adminimal_admin_toolbar": "^1.9",
-    "drupal/apigee_api_catalog": "~1.0",
+    "drupal/apigee_api_catalog": "^1.0@beta",
```

Comment:
Why change this? The ~1.0 will pull in latest instead of being tied to beta versions

Reply:
Fixed this. Thanks.